lib/pengine/container.c

```diff
@@ -530,51 +531,35 @@ create_remote_resource(
             CRM_ASSERT(remote_id_conflict(id, data_set) == FALSE);
         }
 
-        xml_remote = create_resource(id, "pacemaker", "remote");
-
-        /* Abandon our created ID, and pull the copy from the XML, because we
-         * need something that will get freed during data set cleanup to use as
-         * the node ID and uname.
+        /* Using "#uname" as the server name when the connection does not have
+         * its own IP is a hack that allows nested remotes (i.e. a bundle
+         * running on a remote node).
          */
-        free(id);
-        id = NULL;
-        uname = ID(xml_remote);
-
-        xml_obj = create_xml_node(xml_remote, "operations");
-        crm_create_op_xml(xml_obj, uname, "monitor", "60s", NULL);
-
-        xml_obj = create_xml_node(xml_remote, XML_TAG_ATTR_SETS);
-        crm_xml_set_id(xml_obj, "%s-attributes-%d", data->prefix, tuple->offset);
+        connect_name = (tuple->ipaddr? tuple->ipaddr : "#uname");
 
-        if(tuple->ipaddr) {
-            crm_create_nvpair_xml(xml_obj, NULL, "addr", tuple->ipaddr);
-        } else {
-            // REMOTE_CONTAINER_HACK: Allow remote nodes that start containers with pacemaker remote inside
```

Comment:
That was a keyword that is also found elsewhere in the code.
Highly recommend you put it back because a change to one place needs to be reflected in the other